conf/pf.conf.defaults

```diff
@@ -883,10 +883,10 @@ pfdns_processes=4
 # Amount of pffilter processes to start
 pffilter_processes=4
 #
-# advanced.update_iplog_with_accounting
+# advanced.update_ip4log_with_accounting
 #
 # Use the information included in the accounting to update the iplog
-update_iplog_with_accounting=disabled
+update_ip4log_with_accounting=disabled
```

Comment:
Seems this was renamed by mistake, since you didn't change the places where it is used